js/ui/overlay/ui.overlay.js

```diff
@@ -290,11 +289,11 @@ const Overlay = Widget.inherit({
 
         this._toggleViewPortSubscription(true);
         this._initHideTopOverlayHandler(this.option('hideTopOverlayHandler'));
+        this._scrollHandler = (e => { this._targetParentsScrollHandler(e); });
```

Comment:
If we use it only inside of `_toggleParentsScrollSubscription` shouldn't we cache both handler and "parents" there as single object?

Something like
```
this._parentScrollSubsciptionInfo= {
    prevTargets: ...,
    targets: ...,
    handler: ...
}

```